dev-util/intel-ocl-sdk/intel-ocl-sdk-18.1.0.015.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit rpm
+
+DESCRIPTION="Intel's implementation of the OpenCL standard"
+HOMEPAGE="https://www.intel.com/content/www/us/en/developer/articles/tool/opencl-drivers.html#cpu-section"
+SRC_URI="https://registrationcenter-download.intel.com/akdlm/irc_nas/vcp/15532/l_opencl_p_${PV}.tgz"
+
+LICENSE="Intel-SDP"
+SLOT="0"
+KEYWORDS="-* amd64"
+RESTRICT="bindist mirror"
+
+RDEPEND=">=virtual/opencl-3
+	sys-process/numactl"
+DEPEND=""
+PDEPEND="dev-libs/glib
```

Comment:
i know you didn't add this but there is no way any of this should be PDEPEND, there's no circular deps here, they're all pure-RDEPEND as this is a binary package